### ISSUE TITLE
Export result as mzTab?

### ISSUE THREAD

**Issue by jorainer** [rformassspectrometry/MetaboAnnotation#47]
Would be nice to export an `Matched` or `MatchedSpectra` as mzTab?

**Comment by michaelwitting** [comment]
Indeed, that could be really nice. However, a major question might be how to handle multiple `Matched` or similar for the same data set, e.g. one from MS1 and one from MS2 analysis.
